approved/v93k/testflow/mfh.testflow.group/flow_control.tf

```diff
@@ -3764,6 +3794,16 @@ test_flow
     else
     {
     }
+    for @index = 0; @index < @loop ; @index = @index + 1; do
+    test_number_loop_increment = 0
```

Comment:
What's this test_number_loop_increment ?

Reply:
@info-rchitect is this something that just needs to be removed from here?